src/components/input/input.svelte

```diff
@@ -164,7 +156,7 @@
   </slot>
   <slot slot="label" />
 </FormItem>
-{#if showErrors && hasErrors}
+{#if showErrors && $$slots.errors}
```

Comment:
I don't think we should need the `$$slots.errors` here - it's like saying, if we don't have the slot, don't show the slot (which won't happen anyway)
```suggestion
{#if showErrors}
```

Reply:
Ah, yes thanks for pointing that out, makes sense